contracts/sdjwt-verifier/src/msg.rs

```diff
@@ -0,0 +1,76 @@
+use avida_common::types::{
+    RegisterRouteRequest, RouteId, RouteVerificationRequirements, VerfiablePresentation,
+};
+use cosmwasm_schema::QueryResponses;
+use cosmwasm_std::Binary;
+
+use crate::types::{InitRegistration, UpdateRevocationListRequest};
+
+// Contract instantiation parameters
+#[cosmwasm_schema::cw_serde]
+pub struct InstantiateMsg {
+    pub max_presentation_len: usize,
+    pub init_registrations: Vec<InitRegistration>,
+}
+
+// Execute messages
+#[cosmwasm_schema::cw_serde]
+pub enum ExecuteMsg {
```

Comment:
these can be moved to the package so to act as "interface". 

Please look at https://github.com/CosmWasm/cw-plus/tree/main/packages/cw20 and how the packages are used in the contract implementations.

Reply:
Good point, done https://github.com/nymlab/AVIDA/pull/19/commits/19c8202f0ff87688379c9a4b56fb01451014e6ed